### DISCUSSION TITLE
PaddleLabel-ML被PaddleLabel调用，出现405异常

### DISCUSSION THREAD

**Discussion by whwususu** [PaddlePaddle/PaddleOCR#14141] [Q&A]
PaddleLabel配置完PaddleLabel-ML后，只要点击推理救护报405
![image](https://github.com/user-attachments/assets/7265aa20-a08b-482b-aa88-c918cebf3000)
![image](https://github.com/user-attachments/assets/3d39b5ab-dad5-46ac-b65c-ee9b9ea49ead)
![image](https://github.com/user-attachments/assets/40e22806-6e92-4564-82ad-fc11328db988)
除了后端running正常，其它的载入和推理都会报错，PaddleLabel-ML会打印如下异常
![image](https://github.com/user-attachments/assets/7a960d2e-8aed-4e2d-a71c-6a4b3c33dc93)

**Comment by GreatV** [comment]
@Liyulingyue 请张师傅解答一下

**Comment by Liyulingyue** [comment]
PaddleLabel已经长期没有维护了，依赖库有点不适配。暂时没有已知的解决方案。

**Reply by whwususu** [reply]
那这个项目还会继续维护吗？不维护就不敢用啊

**Reply by Liyulingyue** [reply]
我曾经想维护一下，但发现已经有很多其他同类的软件实现这些功能了。所以暂时处于没人维护的状态。

**Reply by whwususu** [reply]
您有不有合适的bs类似软件推荐，labelme是cs的。我们主要想要的是用我们自己的模型，实现自动标注，有问题的再修改，再学习。

**Reply by Liyulingyue** [reply]
我对这个不太了解了

**Reply by whwususu** [reply]
您能否把返回值格式给我发下么，我这边跑不起来。我会按照返回值的格式，组装数据，让paddlelabel显示出来，谢谢！

**Reply by Liyulingyue** [reply]
咨询了一下其他同学，ML模块应该是一直没修过，所以压根跑不起来。此外，我这边也不太清楚接口情况

**Reply by whwususu** [reply]
还有个问题问您下，这个打标的设计器，是否是什么插件，能否给个名字么

**Reply by gouzil** [reply]
如果问的是前端绘制, 可以看看源码中的 [PPDrawTool](https://github.com/PFCCLab/PaddleLabel-Frontend/tree/develop/src/components/PPDrawTool) 和 [PPStage](https://github.com/PFCCLab/PaddleLabel-Frontend/tree/develop/src/components/PPLabelPage/PPStage/index.tsx), 插件可以看看 [Konva](https://konvajs.org/)

**Comment by Flyuove** [comment]
数据集与ML 安装路径不要包含中文，可以避免问题
